### DISCUSSION TITLE
IE11 crashing with 6.08 version

### DISCUSSION THREAD

**Discussion by rvision** [react-hook-form/react-hook-form#2354] [Q&A]
It reports syntax error on this line:

`const parseErrorSchema = (error, validateAllFieldCriteria) => ...`

which is the code of the resolvers. I assume this needs some transpiling since IE doesn't know about const keyword... but I have no idea how to setup this :(

**Comment by bluebill1049** [comment]
are you using IE version?

**Reply by rvision** [reply]
For hook-form I am using IE version. I have an alias setup:

`"react-hook-form": "react-hook-form/dist/index.ie11"`

Do resolvers need something similar?

**Reply by bluebill1049** [reply]
looks like resolver needs an IE 11 version as well :( you want to raise an issue there?

**Reply by rvision** [reply]
Raised: https://github.com/react-hook-form/resolvers/issues/35
